plugins/calibre.koplugin/main.lua

```diff
@@ -414,4 +421,52 @@ function Calibre:getWirelessMenuTable()
     return t
 end
 
+function Calibre:getParserMenuTable()
+    return {
+        {
+            text = _("Automatic"),
+            checked_func = function()
+                return G_reader_settings:hasNot("calibre_json_parser")
+            end,
+            callback = function()
+                G_reader_settings:delSetting("calibre_json_parser")
+            end,
+            hold_callback = function()
+                UIManager:show(InfoMessage:new{
+                    text = _("The program will decide based on the size of the JSON file. Recommended"),
+                })
+            end,
```

Comment:
Err, couldn't those simply be a `help_text`? instead of reimplementing part of that manually? ^^.

Reply:
Lol, I didn't know that.

Reply:
I won't be the one blaming you for not being at ease with our UI widgets, it's really not my area of expertise either ;).